src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/controller/ServiceClientHelper.java

```diff
@@ -104,4 +105,27 @@ public BadRequestException wrapInBadRequestException(
         return new BadRequestException(INVALID_SERVICE_CLIENT_ID + e.getServiceClientIdentifier(),
                 new ErrorDeviation(ERROR_INVALID_SERVICE_CLIENT_ID));
     }
+
+    /**
+     * Sort a list of ServiceClient objects using member name / group description as the primary sort key, and client id
+     * as the secondary sort key.
+     * @param clients
+     */
+    public void sortServiceClientsList(List<ServiceClient> clients) {
+        clients.sort((c1, c2) -> {
```

Comment:
I am happy to see that sorting is a separate feature, and **not** e.g. included in `ServiceClientConverter.convertServiceClientDtos`. I think it keeps the role of converters more clear and retains the possibility of only converting without changing the order.